src/main/java/org/opentripplanner/api/common/RoutingResource.java

```diff
@@ -749,20 +747,6 @@ protected RouteRequest buildRequest(MultivaluedMap<String, String> queryParamete
         setIfNotNull(allowedVehicleRentalNetworks, rental::setAllowedNetworks);
         setIfNotNull(bannedVehicleRentalNetworks, rental::setBannedNetworks);
       }
-      {
-        var parking = journey.parking();
-        setIfNotNull(
-          useVehicleParkingAvailabilityInformation,
-          parking::setUseAvailabilityInformation
-        );
-
-        parking.setFilter(
-          new VehicleParkingFilterRequest(
-            new TagsFilter(bannedVehicleParkingTags),
-            new TagsFilter(requiredVehicleParkingTags)
-          )
-        );
-      }
```

Comment:
I'm in favour of it but I want to clarify that it's the intention to remove these features from the REST API. Is it?

Reply:
I don't think they are removed from the REST API, they are just handled in a different file (https://github.com/opentripplanner/OpenTripPlanner/blob/df56e77849fee2f0c18159aa759fbd0d49a1d0a5/src/main/java/org/opentripplanner/api/common/RequestToPreferencesMapper.java) (although I haven't tested through the REST API yet).

Reply:
Can you test if the REST API still works as intended?

Reply:
No, I will not test it as I want it to go away and I don't care if it's broken.